### ISSUE TITLE
Error L.CRS is undefined

### ISSUE THREAD

**Issue by Lawiss** [thedirtyfew/dash-leaflet#175]
Hello,

I am trying to plot markers on map using dash-leaflet v0.1.25rc1. The problem is that i get a Javascript error "L.CRS is undefined". 
The code that generate the map is :
```py
res_div = html.Div(
        [
            dl.Map(
                [
                    dl.TileLayer(),
                    dl.GeoJSON(url="/assets/data/data_geojson.json", zoomToBounds=True),
                ],
            ),
        ]
    )
```
I verified the geojson file and it is fine, i've also tested (https://github.com/openlayers/openlayers/blob/main/examples/data/geojson/point-samples.geojson)[another geojson] without success.
I am missing something ?

**Comment by emilhe** [comment]
No, I think the code looks OK. Do you get the same error if you use the data from the docs?

Otherwise, one guess might be due to the crs entry in the data,

"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:OGC:1.3:CRS84" }

**Comment by emilhe** [comment]
Could you try the 0.1.23 release? It sounds like a problem with the environment - or a problem with the dash-leaflet build

**Comment by Lawiss** [comment]
The error remains also with 0.1.23 release (I had to downgrade protobuf to 3.18.3 to make it works). I tried downgrading also dash to 2.6 without success. Maybe it is a problem of python version ? My env is currently using python 3.10.
